File: src/main/java/com/claudio_personagens/bookingcharactersapi/domain/Booking.java
package com.claudio_personagens.bookingcharactersapi.domain;

import com.claudio_personagens.bookingcharactersapi.base.AbstractBaseEntity;
import lombok.*;
import org.springframework.beans.factory.annotation.Required;

import javax.persistence.Column;
import javax.persistence.Entity;
import java.time.LocalDate;

@Data
@Entity
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Booking extends AbstractBaseEntity {
    private Client client;
    private CharacterGroup characterGroup;
    private LocalDate bookingAt;
}

File: src/main/java/com/claudio_personagens/bookingcharactersapi/domain/Booking.kt
package com.claudio_personagens.bookingcharactersapi.domain

import com.claudio_personagens.bookingcharactersapi.base.BaseEntity
import java.time.Instant
import javax.persistence.Entity
import javax.persistence.OneToOne

@Entity
data class Booking (
    @OneToOne
    val client: Client?,

    @OneToOne
    val characterGroup: CharacterGroup?,
    val bookingAt: Instant?
): BaseEntity()